reference/docs-conceptual/community/contributing/powershell-style-guide.md

```diff
@@ -0,0 +1,492 @@
+---
+title: PowerShell-Docs style guide
+description: This article provides the rules of style for writing PowerShell documentation.
+ms.date: 03/05/2020
+ms.topic: conceptual
+---
+# PowerShell-Docs style guide
+
+This article provides style guidance specific to the PowerShell-Docs content. This builds on the
+information outlined in the [Overview](overview.md#get-started-writing-docs).
+
+## Product Terminology
```

Comment:
This is the section I was looking for in the other doc.  A link here should suffice.

Reply:
doh